.scalafmt.conf

```diff
@@ -1,3 +1,23 @@
-version = 3.7.17
+version = 3.8.1
 runner.dialect = scala3
 maxColumn = 120
+
+rewrite {
+  rules = [
+    Imports
+  ]
+
+  imports {
+    expand = false
+    sort = ascii
+    groups = [
+      ["gears\\..*"],
+      ["scala\\.meta\\..*", "org\\.scalameta\\..*"],
+      ["sbt\\..*"],
+      ["java.?\\..*"],
+      ["scala\\..*"],
+      ["org\\..*"],
+      ["com\\..*"],
+    ]
```

Comment:
Is it possible to group just by
1. Java & Scala  
2. gears internal imports
3. Everything else

? the current list feels a bit random